scipy/integrate/_quad_vec.py

```diff
@@ -196,6 +196,21 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, li
     ----------
     [1] R. Piessens, E. de Doncker, QUADPACK (1983).
 
+    Examples
+    --------
+    We can compute integrations of a vector-valued function:
+
+    >>> from scipy.integrate import quad_vec
+    >>> import matplotlib.pyplot as plt
+    >>> n = np.linspace(0.0, 10.0)
```

Comment:
Maybe specify also the number of points in linspace to make the example clearer. Also, maybe write `alpha` instead of `n`, because `n` is not integer here.

Reply:
Thank you for review. I fixed it. PTAL.